Classes/System/RestApi/RestApiClient.php

```diff
@@ -151,8 +152,8 @@ protected function createRequestException(RestException $e, $requestMethod, $req
      */
     protected function getRestlerBuilder()
     {
-        $objectManager = GeneralUtility::makeInstance('TYPO3\\CMS\\Extbase\\Object\\ObjectManager');
-        return $objectManager->get('Aoe\\Restler\\System\\Restler\\Builder');
+        $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
```

Comment:
ObjectManager->get is deprecated in TYPO3 10, and will be removed in TYPO3 11. 
So please use the GeneralUtility::makeInstance() directly for the RestlerBuild::class